src/types/PlatformProviderResponse.ts

```diff
@@ -0,0 +1,6 @@
+export interface PlatformProviderResponse {
+  externalId: string;
+  platformProvider: string;
+  data: Record<string, unknown>;
+  createdAt: Date;
```

Comment:
FWIW — and this may just be specific to _my_ career/education and not universal — I _immediately_ associate `createdAt`/`updatedAt` specifically and only with local table insert/update timestamps. In fact if it ever referred to something else (like the remote created-at time of the fetched object), I'd object on those grounds.

Reply:
I figured it out and given @reefdog feedback there is no need to waste any more time on this.

Reply:
@bickelj Whoops, you're right, I hadn't noticed that! I _still_ connect those terms to those concepts, but I was also considering them in a table context. Still, sounds like you're okay with it so no need for me to muse further.